proxy/src/control/discovery.rs

```diff
@@ -598,7 +637,94 @@ where T: HttpService<RequestBody = BoxBody, ResponseBody = RecvBody>,
     }
 }
 
-// ===== impl RxError =====
+// ===== impl DstLabels ====
+
+impl DstLabels {
+
+    #[inline]
+    fn new(set_labels: &HashMap<String, String>,
+           addr_labels: &HashMap<String, String>)
```

Comment:
unclear to me what `set_labels` are...  should this constructor just take an iterator of pairs?